xml/src/test/resources/configs/docs/getting-started.xml

```diff
@@ -42,7 +43,7 @@
   <!-- end::gettingStarted[] -->
 
   <!-- tag::expiry[] -->
-  <cache alias="foo">
```

Comment:
Yes. Because they were moved somewhere else to match the documentation. I didn't want to do that here.